cmd/README.md

```diff
@@ -17,13 +17,22 @@ For additional details, please refer to the [commatrix documentation](https://gi
 
 ---
 
-## Running
-build the code locally and install it in `/usr/local/bin`:
+## Install
+This plugin can be installed with the [Krew](https://sigs.k8s.io/krew) plugin manager:
+
+```sh
+wget https://github.com/openshift-kni/commatrix/blob/main/cmd/commatrix-krew.yaml
```

Comment:
Since the README is in the repo no need to wget, someone reading this have the repo cloned

Reply:
right